cli/cmd/generate_azure.go

```diff
@@ -354,7 +354,7 @@ func initGenerateAzureTfCommandFlags() {
 	generateAzureTfCommand.PersistentFlags().BoolVar(
 		&GenerateAzureCommandState.CreateAdIntegration,
 		"ad_create",
-		true,
+		false,
```

Comment:
Does the config/activity log integration create the ad application if this is set to false and no alternative is supplied by the user?

Reply:
No they get an error if this is not set and they do not give details of an existing active directory integration

Reply:
Based on our conversation via slack, I personally prefer this defaulted to `true`

Reply:
Giving it some thought I think that you are right Ross, I will leave it as it is currently